app/assets/stylesheets/trek.scss

```diff
@@ -0,0 +1,20 @@
+.treks {
+  background-color:scale-color($white, $lightness: -5%);
+	-moz-border-radius:3px;
+	-webkit-border-radius:3px;
+	border-radius:3px;
+	display:inline-block;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered background-color, -moz-border-radius, -webkit-border-radius, border-radius, cursor, display, margin-bottom, padding, text-decoration<br>Colon after property should be followed by one space